pkg/version/version.go

```diff
@@ -19,10 +22,24 @@ var (
 	}
 )
 
+func cleanBuild(build string) string {
+	const (
+		gitprefix = "$Id: "
+		gitsuffix = " $"
+	)
+	if strings.HasPrefix(build, gitprefix) {
+		build = build[len(gitprefix):]
+	}
+	if strings.HasSuffix(build, gitsuffix) {
+		build = build[:len(build)-len(gitsuffix)]
+	}
+	return build
+}
+
 func (v Version) String() string {
 	ver := fmt.Sprintf("Version: %s.%s.%s", v.Major, v.Minor, v.Patch)
 	if v.Metadata != "" {
 		ver += "-" + v.Metadata
 	}
-	return fmt.Sprintf("%s\nBuild: %s", ver, v.Build)
+	return fmt.Sprintf("%s\nBuild: %s", ver, cleanBuild(v.Build))
```

Comment:
Thanks for the change.

Is it possible to change Makefile to include ldflags -X trick to set main.Build by default?
https://stackoverflow.com/questions/11354518/golang-application-auto-build-versioning